go.mod

```diff
@@ -48,6 +48,7 @@ require (
 	github.com/gotesttools/gotestfmt/v2 v2.4.1
 	github.com/grpc-ecosystem/go-grpc-middleware v1.4.0
 	github.com/grpc-ecosystem/grpc-gateway/v2 v2.15.2
+	github.com/hashicorp/go-version v1.6.0
```

Comment:
I see this was already an indirect dep, but how critical is it to add as a full dep? Is it non-trivial to compare versions without it?

Reply:
could you elaborate a bit more? I'm not sure what you mean by indirect dep and full dep.

Reply:
We use `Masterminds/semver` in app to do comparisons between versions [github link](https://github.com/Masterminds/semver) I think using the same library could be a good idea (especially if the cli eventually needs to parse registry item versions).

Reply:
used `Masterminds/semver` for the parsing and comparison in latest impl